### PR TITLE
Issue 133 - License page edits

### PR DESCRIPTION
**Pull request by netsaml**
Converted gerunds to imperative verbs.

### CODE REVIEW

File: content/en/licenses.md [hunk 1]
```diff
@@ -25,14 +25,14 @@ Hopefully you will help others find our project using a statement like:
 
 **This documentation is based on templates from [The Good Docs Project](https://thegooddocsproject.dev/).**
 
-That way, others can also benefit from our project as you are. :)
+That way, others can also benefit from our project as you are. :smile:
```
Comment (barbaricyawps) [inline content/en/licenses.md:28]: Let's convert that back to the old school emoji. I guess Hugo doesn't render it, which makes me sad!

**Review by barbaricyawps — CHANGES_REQUESTED** [review]

File: content/en/licenses.md (hunk 1 above)
Comment (barbaricyawps) [inline content/en/licenses.md:28]:
```suggestion
That way, others can also benefit from our project as you are. :)
```

**Review by barbaricyawps — CHANGES_REQUESTED** [review]

**Comment by deanna-cs** [comment]
PR looks great!

**Review by barbaricyawps — APPROVED** [review]

**Review by deanna-cs — APPROVED** [review]

**Comment by barbaricyawps** [comment]
@netsaml , thanks for making those changes! Congratulations on your first pull request!

File: content/en/licenses.md (hunk 1 above)
Comment (zivbk1) [reply content/en/licenses.md]: > Let's convert that back to the old school emoji. I guess Hugo doesn't render it, which makes me sad!

Hugo should render emojis.

**Pull request merged**